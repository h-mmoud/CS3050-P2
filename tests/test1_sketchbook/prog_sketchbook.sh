#!/bin/bash
sh ../run.sh ../examples/sketchbook.fp