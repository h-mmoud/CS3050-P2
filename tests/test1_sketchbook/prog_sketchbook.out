yes
X = prof_oreo
X = sir_nut